controllers/users.js

```diff
@@ -732,16 +732,36 @@ const addDefaultArchivedRole = async (req, res) => {
  * @param res {Object} - Express response object
  */
 
+const calculatePagination = (pageNumber, totalPages, reqQuery, limitNumber) => {
+  const nextPage = pageNumber < totalPages - 1 ? pageNumber + 1 : null;
+  const prevPage = pageNumber > 0 ? pageNumber - 1 : null;
+
+  return {
+    next: nextPage ? getFilteredPaginationLink(reqQuery, nextPage, limitNumber) : null,
+    prev: prevPage ? getFilteredPaginationLink(reqQuery, prevPage, limitNumber) : null,
+  };
+};
+
 const filterUsers = async (req, res) => {
   try {
     if (!Object.keys(req.query).length) {
       return res.boom.badRequest("filter for item not provided");
     }
+    const { page, limit } = req.query;
+    const pageNumber = parseInt(page) || 0;
+    const limitNumber = parseInt(limit) || 100;
+    const skip = (pageNumber - 1) * limitNumber;
```

Comment:
Do you know if the skip functionality is implemented?

Reply:
Yes, It is implemented

`const users = await dataAccess.retreiveFilteredUsers(req.query, skip, limitNumber);`

Reply:
Can we have some test or proof of skip functionality? Didn't find this functionality in code.

Reply:
@prakashchoudhary07 @ajoykumardas12 added test for skip functionality.